cpp/src/io/orc/aggregate_orc_metadata.cpp

```diff
@@ -171,8 +171,9 @@ std::vector<metadata::stripe_source_mapping> aggregate_orc_metadata::select_stri
 
       // Coalesce stripe info at the source file later since that makes downstream processing much
       // easier in impl::read
-      for (const size_t& stripe_idx : user_specified_stripes[src_file_idx]) {
-        CUDF_EXPECTS(stripe_idx < per_file_metadata[src_file_idx].ff.stripes.size(),
+      for (const auto& stripe_idx : user_specified_stripes[src_file_idx]) {
```

Comment:
Just small nit:
```suggestion
      for (const auto stripe_idx : user_specified_stripes[src_file_idx]) {
```

Reply:
@ttnghia Can you explain why you're suggesting this change?

Reply:
Sorry for late reply (I didn't see this).
A reference is similar to pointer, which is 64 bits. If you use reference for 32 bits number, that's kind of inefficient. Although the compiler can optimize this and generate the same code (for references and without references), it is still logically inefficient.